TestData/Bad.xcodeproj/project.pbxproj

```diff
@@ -96,7 +96,10 @@
 				D2A90D18203219FC00EBA6AA /* ThisGroupIsEmpty */,
 			);
 			path = Bad;
+			indentWidth = 7;
 			sourceTree = "<group>";
+			tabWidth = 7;
+			usesTabs = 1;
```

Comment:
It's going to be hard to know why the test failed, though. Ideally, we'd have these scenarios broken out into more granular tests so we can validate each logic branch (both for file/project settings, as well as the specific whitespace rules)

Reply:
I kept going back and forth on spreading them out so we get a hit on each separate, because of my use of the or logic.  But not 100% sure on how much that needs to be tested out.  I am okay spreading one to each line to show getting a ping on each item.

Reply:
Let's look at it from the perspective of a person adding this tool to their build process.

When you get the error, what does it say? 

When you address the reported error, does the next build succeed?

It seems to me that knowing you had `tabWidth` _and_ `usesTabs` set would be more informative than knowing you had "white space specifiers" set.

The goal would be to make it more likely you'd fix it the first time.

Reply:
Okay.  Alex had said the same thing, and I worried it would be too spammy.  But if that is preferred, I will change to display every one of them found.